news/2018-02-20-community-choice-2017-results.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-02-20 08:00:00 +0000
+---
+
+The results are finally in, and it is time to announce the Community Favourites of 2017!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+After some considerable deliberation as to how to handle the results, we've decided to go with a simple tally of most votes received to determine the winner for this year's vote, in a departure from the tradition of weighting the votes based on a variety of factors.
+
+Given that the vote this year is purely aimed at finding the most popular map from the community among each mode, anything more than that would be unwarranted!
+
+Without further ado, here are the four grand winners from each mode:
+
+## osu!
+
+![](/wiki/shared/news/2018-02-20-community-choice-2017-results/hitorigoto.jpg)
+
+In an overwhelming display of support that should surprise exactly nobody, [Doormat](https://osu.ppy.sh/u/3230571)'s set of [ClariS - Hitorigoto -TV MIX-](https://osu.ppy.sh/s/596704) takes the title for Community Favourite of 2017 with an outstanding **4434** votes, nearly a full thousand more than its nearest competitor.
+
+Riding off the (questionable) success of last year's hit anime Eromanga-sensei, Hitorigoto is a track all about talking to one's self, and this highly popular set certainly managed to get people talking.
+
+Notable runners up were:
+
+[Agnete Kjolsrud - Get Jinxed](http://osu.ppy.sh/s/361740) by [Tarrasky](https://osu.ppy.sh/u/4098393) in 2nd place overall with **3565** votes.
+
+[\*namirin - Koi no Hime Hime Pettanko](http://osu.ppy.sh/s/485056) by [C u r i](https://osu.ppy.sh/u/4746949) in 3rd place overall with **2749** votes.
+
+## osu!taiko
+
+![](/wiki/shared/news/2018-02-20-community-choice-2017-results/renren.jpg)
+
+By the skin of its teeth with only **9** votes between first and second place, [Backfire](https://osu.ppy.sh/u/263110)'s set of [Sara - Ren Ren Ai Ai Cir Cir Cula Cula Tion Tion](https://osu.ppy.sh/s/663337) snags the title of Community Favourite 2017 for osu!taiko with **117** votes.
+
+Unlike the track itself, we're definitely not stuttering.
+
+Notable runners up were:
+
+[D(ABE3) - MANIERA](http://osu.ppy.sh/s/400761) by [\_yu68](https://osu.ppy.sh/u/6170507) with **108** votes.
+
+[Dunderpatrullen - We Are Number One (Remix)](https://osu.ppy.sh/s/547301) by [Midnaait](https://osu.ppy.sh/u/1506011) with **104** votes.
+
+## osu!catch
+
+![](/wiki/shared/news/2018-02-20-community-choice-2017-results/onthefm.jpg)
+
+Fiercely competitive all the way to the end, osu!catch had some of the most robust voting of all the gamemodes so far, a few big maps fighting for the lion's share of the vote.
+
+In the end, [Ascendance](https://osu.ppy.sh/u/2931883)'s set of [C\-Show - On the FM](https://osu.ppy.sh/s/609679) triumphed over the others with **249** votes, and walks away with the title of Community Favourite 2017 for osu!!catch.
```

Comment:
Two exclamation marks on "osu!!catch", please curb your excitement for our fair fruitmode